teps/0131-tekton-conformance-policy.md

```diff
@@ -0,0 +1,251 @@
+    - [3. Do we only include GA primitives in the conformance policy?](#3-do-we-only-include-ga-primitives-in-the-conformance-policy)
+    - [4. Should we define the Conformance Policy per Tekton Services?](#4-should-we-define-the-conformance-policy-per-tekton-services)
+  - [Open Questions](#open-questions)
+    - [1. How long do we want the policy update notice to be ahead of?](#1-how-long-do-we-want-the-policy-update-notice-to-be-ahead-of)
+    - [2. Do we consider the previous Conformance version valid after the new version is released?](#2-do-we-consider-the-previous-conformance-version-valid-after-the-new-version-is-released)
+    - [3. What to do with field deprecation?](#3-what-to-do-with-field-deprecation)
+- [Implementation Pull Requests](#implementation-pull-requests)
+- [References](#references)
+<!-- /toc -->
+
+## Summary
+
+This TEP proposes **Tekton Conformance Policy** which lays out the detailed
+requirements that Tekton vendors need to satisfy to claim Tekton `<Primitive Type>`
+Conformance on a Service that the vendor provides. It guarantees portability
+of Tekton Conformant Primitives in the Tekton ecosystem and interoperability
+of Tekton Conformant Primitives across Tekton vendors. It defines:
+* **Conformance Primitive Types**: vendors can claim Conformance of a ***Tekton Primitive Type*** on the services they provide.
+* **Conformance Requirements**: requirements of claiming the conformance:
+  * Tekton APIs to be implemented.
+  * Syntactical and semantical requirements of each API field.
+* **Conformance Policy Versioning**: use versioning to keep track of updates of the policy.
+* **Conformance Policy Update Procedure**: standard procedure to update the policy.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals and non-goals of
+this TEP. Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports][experience reports]
+to demonstrate the interest in a TEP within the wider Tekton community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+Starting from Tekton release v0.43.0, Tekton has released **V1 CRDs** including `Task`, `TaskRun`, `Pipeline`, `PipelineRun`. As our APIs reaching stable state, we want to explore how to build a better Tekton ecosystem that both **Tekton users** and **Tekton vendors** can benefit from.
+
+![Benefits of Tekton Conformance Policy](/teps/images/0131-benefits-of-tekton-conformance-policy.png)
+
+The left side is the world we are in today, for **Tekton users** who wants to use Tekton, they have to:
+- manage their own kubernetes cluster
+- install Tekton Pipelines on the cluster
+- write YAMLs to define their CI/CD pipelines
+
+The right side is the world we want to live after we have the conformance policy:
```

Comment:
nit: we already have vendors that build services on top of tekton (e.g. openshift pipelines), and we aren't trying to remove the self-hosted option

Reply:
Great catch, updated!

Reply:
I still think this could be reworded. If we're saying right now, users have to manage their own k8s clusters, and we hope to live in a world where vendors provide tekton services, it sounds like there won't be a self hosted option. Maybe you can say "users can port their pipelines between conformant vendor offerings and self-hosted clusters"?